types/bluebird/index.d.ts

```diff
@@ -194,8 +194,8 @@ declare class Bluebird<R> implements PromiseLike<R>, Bluebird.Inspection<R> {
    *
    * Alias `.caught();` for compatibility with earlier ECMAScript version.
    */
-  caught(onReject?: (error: any) => R | PromiseLike<R | void> | undefined): Bluebird<R>;
```

Comment:
Can you move those two `caught()` definitions up right below the `catch()` ones in line 57 and make the function signature identical? (Since they're aliases.)
Afterwards I think this PR is ready to be merged. 🙂

Reply:
@segayuu Nevermind: Those two definitions should of course stay where they are.
But the function signature should be the same for both `catch()` and `caught()`. 🙂